woocommerce/woocommerce
# Add common options to the variations options empty state

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43808



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure feature `New product editor` is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure feature `product-variation-management` is enabled under `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper` (WooCommerce Beta Tester plugin) is required
3. Go to `/wp-admin/admin.php?page=wc-admin&path=/add-product`
4. Under the `Variations` tab a new empty state should be shown in the `Variation options` section 
<img width="723" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/4ab31552-2ca0-4053-9f0c-6dd8a83e5928">

5. When a user clicks on the secondary buttons of Add sizes or Add colors, then those attributes will be automatically populated into the input field. 
<img width="959" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/31ecaf06-a0fb-494b-aff7-39be61f0ca9d">

6. If the attribute (Size / Color) has already been added to the store, then it should appear in the dropdown menu below.
7. If the attribute (Size / Color) has not already been added to the store, then it can be created from the dropdown menu below.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
